### PR TITLE
feat: updated the get started layout page

### PR DESCRIPTION
**Pull request by BiswaViraj**
### What change does this PR introduce?

- Updated the get started page
- made the footer absolute position
### Other information (Screenshots)

https://github.com/novuhq/novu/assets/29251776/569e29e8-3e78-4092-90c7-433b88be1292

### CODE REVIEW

File: apps/web/src/components/layout/components/PageContainer.tsx [hunk 1]
```diff
@@ -18,11 +19,18 @@ function PageContainer({
...
 }
 
 export default PageContainer;
+
+const StyledContainer = styled(Container)`
+  overflow-y: auto;
+  border-radius: 0;
+  padding-left: 0;
+  padding-right: 0;
+`;
```
Comment (BiswaViraj) [inline apps/web/src/components/layout/components/PageContainer.tsx:36]: style props didnt accepted the overflowY property, son had to create a styled component.

Without overflow-y: auto, the container was getting cut off at 100vh - 40px
image
adding overflow fixes the issue

File: apps/web/src/pages/quick-start/steps/DigestPreview.tsx [hunk 2]
```diff
@@ -28,8 +27,7 @@ export function DigestPreview() {
     <GetStartedLayout
       header={
         <>
-          <HeaderTitle>Set-up steps to get started</HeaderTitle>
-          <HeaderSecondaryTitle>Quick Start Guide</HeaderSecondaryTitle>
+          <Title>Get started</Title>
```
Comment (ainouzgali) [inline apps/web/src/pages/quick-start/steps/DigestPreview.tsx:30]: Are we sure the titles need to be removed?

**Review by ainouzgali — APPROVED** [review]

File: apps/web/src/pages/quick-start/steps/DigestPreview.tsx (hunk 2 above)
Comment (BiswaViraj) [reply apps/web/src/pages/quick-start/steps/DigestPreview.tsx:30]: Right, it was not supposed to be, reverting it.

Comment (BiswaViraj) [reply apps/web/src/pages/quick-start/steps/DigestPreview.tsx]: done
image


Step 1 stays with the new design:
image

**Pull request merged**